lib/db/schema/patch-082-083.sql

```diff
@@ -0,0 +1,13 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CREATE PROCEDURE `assert_patch_level` (
```

Comment:
Should we version this `assert_patch_level_1`?

Reply:
I don't foresee it changing and it makes the related boilerplate that we add to every migration a little bit less readable, so I'm inclined not to. Happy to do it though if you or others feel strongly about it.